jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -659,8 +662,45 @@ YUI.add('deployment-flow', function() {
         </div>);
     },
 
+    /**
+      Report whether going forward with the deployment is currently allowed.
+
+      @method _deploymentAllowed
+      @returns {Bool} Whether deployment is allowed.
+    */
+    _deploymentAllowed: function() {
```

Comment:
But that block would not work, as it would return false even on legacy juju if you have a model name.